src/protocols/hii_database.rs

```diff
@@ -0,0 +1,18 @@
+// Copyright © 2019 Intel Corporation
+// Copyright (c) Microsoft Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#![allow(unused)]
+
+pub type HiiHandle = *mut core::ffi::c_void;
```

Comment:
I am tempted to recommend moving this to `src/hii.rs`. I know it is defined in the HII-Database protocol, but it is kinda re-used all over the place, and it is only an associated definition. This is debatable, so if you prefer it in the database-protocol, I will not object (alternatively, we can also alias it in both places).